db/migrate/20201110090452_create_drivers.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDrivers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :drivers do |t|
+      t.string :name
```

Comment:
When you validate presence on the model side
```
validates :name, presence: true
```

I suggest to also add DB constraint

```
t.string :name, null: false
```

Reply:
what does the null: false do?

Reply:
`null: false` in a Rails migration tells your database not to accept NULL values.

Reply:
in that case, should i add it whenever i validate presence?

Reply:
pretty much yeah, they go hand in hand for really reliable data